pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -299,6 +301,11 @@ func (k *Kubernetes) UpdateKubernetesObjects(name string, service kobject.Servic
 		template.Spec.Containers[0].WorkingDir = service.WorkingDir
 		template.Spec.Containers[0].VolumeMounts = volumesMount
 		template.Spec.Volumes = volumes
+		cpu, err := resource.ParseQuantity(strconv.FormatInt(service.CPUShares,10))
+		if err != nil {
+			logrus.Error(err)
+		}
```

Comment:
I think this code is all great, but it could be better suited in a separate function similar to how we get the conversion from volumesMount.

This will help make it look prettier by having just:

```
template.Spec.Volumes ...
template.Spec.Containers[0].Resources ...
...
```